### PR TITLE
Fixes Topeka sw issues

### PR DESCRIPTION
**Pull request by ebidel**
R: @robdodson @addyosmani

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -28,15 +28,12 @@
...
+  but styling is off on paper-fabs.
   -->
-  <script>
-    if (!('import' in document.createElement('link'))) {
-      var Platform = {flags: {shadow: true}};
-    }
-  </script>
+  <!-- <script>
+    var WebComponents= {flags: {shadow: true}};
+  </script> -->
```
Comment (robdodson) [inline index.html]: should this be removed?

File: sw.js [hunk 2]
```diff
@@ -64,9 +64,7 @@ this.addEventListener("install", function(e) {
       var resourceUrls = [
         "",
         "?offline",
-        "build.html",
         "build.js",
-        "components/topeka-elements/",
```
Comment (robdodson) [inline sw.js]: why were the two above items removed?

**Comment by robdodson** [comment]
reviewed

File: index.html (hunk 1 above)
Comment (ebidel) [inline index.html]: Probably, but not using the polyfill forces the errors in Chrome and I don't have the knowledge of the polyfill to know what's going on. I left it and the comment for someone in the future and take a look.

File: sw.js (hunk 2 above)
Comment (ebidel) [inline sw.js]: The were causing 404 errors in the sw caching code

**Comment by robdodson** [comment]
LGTM

**Pull request merged**